applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt_adv.c

```diff
@@ -19,6 +19,7 @@ LOG_MODULE_DECLARE(bt_mgmt);
 ZBUS_CHAN_DECLARE(bt_mgmt_chan);
 
 static struct k_work adv_work;
+
```

Comment:
I think no blank lines here is better, these are all static structs regarding adv

Reply:
Hmm, this is the advertising file, so it is kind of expected that all structs are related to adv. I think these newlines help splitting them up "based on function"